pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/graphql_helpers_test.go

```diff
@@ -39,6 +39,38 @@ func TestGetNumPages1(t *testing.T) {
 	assert.Equal(t, expected, num)
 }
 
+func TestAdd100(t *testing.T) {
+	a := 100
+	b := 100
+
+	expected := 200
+
+	num := add(a, b)
+
+	assert.Equal(t, expected, num)
+}
+
+func TestAdd10(t *testing.T) {
+	a := 10
+	b := 10
+
+	expected := 20
+
+	num := add(a, b)
+
+	assert.Equal(t, expected, num)
+}
+
+func TestAdd1(t *testing.T) {
```

Comment:
can we add a negative number with a postive number in the tests? I presume that github won't respond with a negative number

Reply:
👍